terraspace/app/stacks/peer-e2e-testing/README.md

```diff
@@ -0,0 +1,3 @@
+# Readme
+
+Remember to build packer image after making app code changes so they can be reflected in new EC2 instances
```

Comment:
We're not copying the code with shell (like bucket to mirror), so there's no need for this disclaimer